src/pybind/mgr/mgr_util.py

```diff
@@ -10,7 +10,7 @@
 if 'UNITTEST' in os.environ:
     import tests  # noqa
 
-import bcrypt
+import base64
```

Comment:
Same thing, can we avoid base64-encoding stuff by passing binary data instead? We shouldn't pass sensitive (not even secrets, just hints) through the CLI.

Reply:
Done.